jmix-flowui/flowui-kit/src/main/java/io/jmix/flowui/kit/component/main/ListMenu.java

```diff
@@ -385,9 +379,29 @@ protected Details createMenuBarComponent(MenuBarItem menuBarItem) {
 
         menuItemComponent.setContent(menuList);
 
+        menuItemComponent.addOpenedChangeListener(this::menuBarOpenedChangeListener);
+
         return menuItemComponent;
     }
 
+    protected void menuBarOpenedChangeListener(Details.OpenedChangeEvent event) {
```

Comment:
Roleback to prev implementation because event is needed for Detail menu item only and not for its children